### ISSUE TITLE
Netgraph fails when no interfaces available

### ISSUE THREAD

**Issue by jagguli** [qtile/qtile#913]
```
May 11 10:14:02 steven-series9 qtile-session[22026]: During handling of the above exception, another exception occurred:
May 11 10:14:02 steven-series9 qtile-session[22026]: Traceback (most recent call last):
May 11 10:14:02 steven-series9 qtile-session[22026]:   File "/usr/lib/python3.5/site-packages/libqtile/widget/graph.py", line 336, in __init__
May 11 10:14:02 steven-series9 qtile-session[22026]:     self.interface = self.get_main_iface()
May 11 10:14:02 steven-series9 qtile-session[22026]:   File "/usr/lib/python3.5/site-packages/libqtile/widget/graph.py", line 377, in get_main_iface
May 11 10:14:02 steven-series9 qtile-session[22026]:     raise RuntimeError('No valid interfaces available')
May 11 10:14:02 steven-series9 qtile-session[22026]: RuntimeError: No valid interfaces available
May 11 10:14:02 steven-series9 qtile-session[22026]: During handling of the above exception, another exception occurred:
May 11 10:14:02 steven-series9 qtile-session[22026]: Traceback (most recent call last):
May 11 10:14:02 steven-series9 qtile-session[22026]:   File "/usr/lib/python3.5/site-packages/libqtile/confreader.py", line 55, in __init__
May 11 10:14:02 steven-series9 qtile-session[22026]:     config = __import__(os.path.basename(fname)[:-3])
May 11 10:14:02 steven-series9 qtile-session[22026]:   File "/home/steven/.config/qtile/config.py", line 76, in <module>
May 11 10:14:02 steven-series9 qtile-session[22026]:     screens = get_screens(num_monitors, num_groups, groups)
May 11 10:14:02 steven-series9 qtile-session[22026]:   File "/home/steven/.config/qtile/screens.py", line 159, in get_screens
May 11 10:14:02 steven-series9 qtile-session[22026]:     widget.NetGraph(**netgraph_params),
May 11 10:14:02 steven-series9 qtile-session[22026]:   File "/usr/lib/python3.5/site-packages/libqtile/widget/graph.py", line 338, in __init__
May 11 10:14:02 steven-series9 qtile-session[22026]:     self.log.warning(
May 11 10:14:02 steven-series9 qtile-session[22026]:   File "/usr/lib/python3.5/site-packages/libqtile/configurable.py", line 44, in __getattr__
May 11 10:14:02 steven-series9 qtile-session[22026]:     raise AttributeError("%s has no attribute: %s" % (cname, name))
May 11 10:14:02 steven-series9 qtile-session[22026]: AttributeError: NetGraph has no attribute: log
```

**Comment by flacjacket** [comment]
What version of qtile are you running? We have switched our logging over from passing around a log instance to call `self.log` to having a single importable `logger` instance.

**Comment by jagguli** [comment]
I'm using 
0.10.4

Seems to be fixed in develop

**Comment by flacjacket** [comment]
This should also be fixed in 0.10.5, since it's alright in a current release, I'll close this, let us know if you still have problems with this
